StoreTestWPF.Presentation/StoreTestWPF.Presentation.csproj

```diff
@@ -0,0 +1,55 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.WindowsDesktop">
+
+  <PropertyGroup>
+    <OutputType>WinExe</OutputType>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <UseWPF>true</UseWPF>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <None Remove="Resource\Images\Apple_pie.jpg" />
+    <None Remove="Resource\Images\Ciasto_czekoladowo_migdałowe_z_gruszkami.jpg" />
+    <None Remove="Resource\Images\favicon.png" />
+    <None Remove="Resource\Images\Fraisier.jpg" />
+    <None Remove="Resource\Images\Kokosowy_sernik_królewski.jpg" />
+    <None Remove="Resource\Images\Tarta_z_mascarpone_rabarbarem_i_truskawkami.jpg" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="EntityFramework" Version="6.4.4" />
```

Comment:
Do we really need EF on Presentation layer?

Reply:
Fixed